src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -29,26 +27,32 @@ export default class Checkbox extends React.PureComponent<Props> {
     };
 
     handleClick = () => this.input.click();
+
     handleInputClick = (event: Event) => event.stopPropagation();
+
     setInput = (input: ElementRef<'input'>) => this.input = input;
 
     render() {
-        const classNames = classnames({
-            [checkboxStyles.checkbox]: true,
-            [checkboxStyles[this.props.skin]]: true,
-            [this.props.className]: !!this.props.className,
-        });
+        const {
+            checked,
+            className,
+        } = this.props;
+        const checkboxClass = classNames(
```

Comment:
Nevermind, your way seems to be the recommended one in [their README](https://github.com/JedWatson/classnames#usage-with-reactjs).